Gemfile.lock

```diff
@@ -1,3 +1,11 @@
+GIT
+  remote: https://github.com/faker-ruby/faker.git
+  revision: 6aecb58a34283f58f40cfb05e2245c55e0439190
+  branch: master
+  specs:
+    faker (2.22.0)
+      i18n (>= 1.8.11, < 2)
+
```

Comment:
Is this automatically generated when you install the Faker gem? If not, what is this?